HSLdevcom/helmet-ui
# #185: Show all scenario runs in a list instead of only the current run

How to test:

`npm install` to install all dependencies
`npm start` - launch app locally

Activate some scenarios and start them. The scenario runs should be displayed in a list separately instead of only showing the current run's graph. Each graph also has the link to logs under it exactly as before.
